### ISSUE TITLE
Update to Knative v1.11

### ISSUE THREAD

**Issue by christophd** [apache/camel-k#4687]
Knative v1.11 is available (Knative golang API version is v0.38)

**Comment by squakez** [comment]
Yes, it depends on #4612 - I was having a look but get hooked on something else. Feel free to have a look yourself.
